stress/.travis.yml

```diff
@@ -0,0 +1,90 @@
+dist: trusty
```

Comment:
Sorry! But, I still don't understand. Why are we adding .travis.yml inside stress folder? As discussed, the make steps to build stress image alone needs to be added in the already present .travis.yml in test-tools repo root. @rajdas98 can you help @uditgaurav with this please?